nix/hm-module.nix

```diff
@@ -46,6 +46,15 @@ in {
             default = false;
             description = "Prefer GeoClue 2 over manual location configuration.";
           };
+          timeout = lib.mkOption {
+            type = lib.types.int;
```

Comment:
Just a small observation that this could be changed to `with lib.types; nullOr path;` to support using the default value.

Thanks for the update!

Reply:
Right!